perftests/standalone-enc/fd_drain.h

```diff
@@ -0,0 +1,24 @@
+#ifndef FD_DRAIN_H_
+#define FD_DRAIN_H_
+
+#include <sys/types.h>
+
+/* The smallest this can be is PCRYPT_ESZ (which is 1060). */
+#define FD_DRAIN_MAX_SIZE 16384
+
+/**
+ * fd_drain(fd_p):
+ * Drain bytes from the file descriptor ${*fd_p} -- passed as an (int *) -- as
+ * quickly as possible.
+ */
+int fd_drain(void *);
```

Comment:
My original intent was only to expose `fd_drain_fork()`, but `standalone_transfer_noencrypt.c` uses threads rather than processes, so it needs to call `fd_drain()` directly.

Reply:
I'm not a fan of the "pass a pointer to the file descriptor, cast to void *" API here.  How about `int fd_drain(int)` and `static int fd_drain_internal(void *)` which gets called by both of the externally-visible functions?